.github/workflows/main.yml

```diff
@@ -20,6 +20,6 @@ jobs:
   test:
     name: Ruby
     needs: rubocop
-    uses: theforeman/actions/.github/workflows/foreman_plugin.yml@v0
+    uses: theforeman/actions/.github/workflows/foreman_plugin.yml@dbseed
```

Comment:
should be dropped once https://github.com/theforeman/actions/pull/15 is merged

Reply:
That's merged.